pkg/platform/platform.go

```diff
@@ -35,15 +35,15 @@ func InfraStatus(client cnoclient.Client) (*bootstrap.InfraStatus, error) {
 		APIServers:             map[string]bootstrap.APIServer{},
 	}
 
-	// If we use hypershift, the components need proxy settings. In selfhosted, they
-	// are always local to the API so there it is never needed.
-	if res.ControlPlaneTopology == configv1.ExternalTopologyMode {
-		proxy := &configv1.Proxy{}
-		if err := client.Default().CRClient().Get(context.TODO(), types.NamespacedName{Name: "cluster"}, proxy); err != nil {
-			return nil, fmt.Errorf("failed to get proxy 'cluster': %w", err)
-		}
-		res.Proxy = proxy.Status
+	// If we use hypershift, the components need proxy settings. In selfhosted,
```

Comment:
I don't think that comment makes sense here.
The code below just retrieves the proxy config.